src/main/java/hibernate/tournament/controller/TournamentController.java

```diff
@@ -0,0 +1,47 @@
+package hibernate.tournament.controller;
+
+import hibernate.tournament.model.Team;
+import hibernate.tournament.repository.TeamRepository;
+import hibernate.tournament.service.TournamentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class TournamentController {
+
+    private TournamentService tournamentService;
+    private TeamRepository teamRepository;
+
+    @Autowired
+    public TournamentController(TournamentService tournamentService, TeamRepository teamRepository){
+        this.tournamentService = tournamentService;
+        this.teamRepository = teamRepository;
+    }
+
+
+    @GetMapping("/launch")
+    public String launchGame() throws Exception{
+        tournamentService.launch();
+
+        return "launch";
+    }
+
+    @PostMapping("/addTeam")
```

Comment:
Example of good pattern:
`GET /teams/{id}` get team by id
`GET /teams` get all teams
`POST /teams` create new team
`DELETE /teams/[id]` delete team by id

You see? endpoint names are easy to predict and use. Endpoint name barely express action, HTTP method expresses action.